### PR TITLE
Fix: Pillow error when uploading RGBA image (#3325)

### PR DESCRIPTION
**Pull request by regnarg**
Signed-Off-By: Filip Štědronský <g@regnarg.cz>

### Pull Request Checklist

* [x] Pull request is based on the develop branch
* [x] Pull request includes a [changelog file](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.rst#changelog)
* [x] Pull request includes a [sign off](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.rst#sign-off)

### CODE REVIEW

**Comment by richvdh** [comment]
oh, some of the CI is failing because `black` has had a new release. `black` is pinned on develop - if you merge `develop` into your branch it will fix some of the failures.

**Comment by richvdh** [comment]
@regnarg thanks. it's still complaining about the one-line `if` though: https://buildkite.com/matrix-dot-org/synapse/builds/5285#7b2503e7-02b3-4b73-92c9-92be0f2cb88f

**Comment by regnarg** [comment]
It seems the CI check got stuck (for several days)

**Comment by richvdh** [comment]
sorry, it needs running manually

File: synapse/rest/media/v1/thumbnailer.py [hunk 1]
```diff
@@ -129,5 +129,8 @@ def crop(self, width, height, output_type):
 
     def _encode_image(self, output_image, output_type):
         output_bytes_io = BytesIO()
-        output_image.save(output_bytes_io, self.FORMATS[output_type], quality=80)
+        fmt = self.FORMATS[output_type]
+        if fmt == 'JPEG':
```
Comment (richvdh) [inline synapse/rest/media/v1/thumbnailer.py]:
```suggestion
        if fmt == "JPEG":
```

Comment (tcyrus) [reply synapse/rest/media/v1/thumbnailer.py]: It looks like this change was made, so this request can be marked as resolved.

**Pull request merged**